pkg/server/exec/exec.go

```diff
@@ -0,0 +1,106 @@
+package exec
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/luizalabs/teresa/pkg/server/app"
+	"github.com/luizalabs/teresa/pkg/server/auth"
+	"github.com/luizalabs/teresa/pkg/server/database"
+	"github.com/luizalabs/teresa/pkg/server/spec"
+	"github.com/luizalabs/teresa/pkg/server/storage"
+)
+
+type Operations interface {
+	Command(user *database.User, appName string, command ...string) (io.ReadCloser, <-chan error)
+	CommandBySpec(podSpec *spec.Pod) (io.ReadCloser, <-chan error)
+}
```

Comment:
Maybe `RunCommand` and `RunPod` (`PodRun` ?)

Reply:
I was thinking in the complete command call (with the pkg), e.g. `exec.Command` (instead of `exec.RunCommand`), but yes, I have some doubts... `exec.RunCommand` sounds good to me.